packages/paste-core/components/help-text/src/HelpText.tsx

```diff
@@ -17,63 +16,70 @@ export const HelpTextVariants = {
 // eslint-disable-next-line @typescript-eslint/no-redeclare
 export type HelpTextVariants = ValueOf<typeof HelpTextVariants>;
 
-export interface HelpTextProps extends React.HTMLAttributes<HTMLDivElement> {
+export interface HelpTextProps extends React.HTMLAttributes<HTMLDivElement>, Pick<BoxProps, 'element'> {
   marginTop?: 'space0';
   variant?: HelpTextVariants;
 }
 
-const HelpText = React.forwardRef<HTMLDivElement, HelpTextProps>(({marginTop, children, variant, ...props}, ref) => {
-  let icon = null;
-  switch (variant) {
-    case HelpTextVariants.ERROR:
-      icon = (
-        <Box flexShrink={0}>
-          <ErrorIcon color="colorTextError" decorative size="sizeIcon20" />
-        </Box>
-      );
-      break;
-    case HelpTextVariants.ERROR_INVERSE:
-      icon = (
-        <Box flexShrink={0}>
-          <ErrorIcon color="colorTextErrorWeak" decorative size="sizeIcon20" />
-        </Box>
-      );
-      break;
-    default:
-      break;
-  }
+const HelpText = React.forwardRef<HTMLDivElement, HelpTextProps>(
+  ({marginTop, children, variant, element = 'HELP_TEXT', ...props}, ref) => {
+    let icon = null;
+    switch (variant) {
+      case HelpTextVariants.ERROR:
+        icon = (
+          <Box flexShrink={0}>
+            <ErrorIcon color="colorTextError" decorative size="sizeIcon20" />
+          </Box>
+        );
+        break;
+      case HelpTextVariants.ERROR_INVERSE:
+        icon = (
+          <Box flexShrink={0}>
+            <ErrorIcon color="colorTextErrorWeak" decorative size="sizeIcon20" />
+          </Box>
+        );
+        break;
+      default:
+        break;
+    }
 
-  let textColor = 'colorTextWeak' as TextColor;
-  if (variant === 'error') {
-    textColor = 'colorTextError';
-  } else if (variant === 'error_inverse') {
-    textColor = 'colorTextErrorWeak';
-  } else if (variant === 'inverse') {
-    textColor = 'colorTextInverseWeak';
-  }
+    let textColor = 'colorTextWeak' as TextColor;
+    if (variant === 'error') {
```

Comment:
On line 28 we use constants to check the variant, but here we're using strings. Can we use the constants again?

Reply:
Yeah, completely refactored this to be a little safer